currency_rate_update_VN_VCB/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# © 2018 Komit <http://komit-consulting.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Currency Rate Update - VCB',
+    'summary': 'Currency Rate Update Vietcombank service',
+    'version': '11.0.0.0.0',
```

Comment:
Should be: 
    "version": "11.0.1.0.0",

https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py